docs/about/faq.rst

```diff
@@ -40,7 +40,7 @@ Variant descriptions will be accepted into CIViC provided there is evidence link
 
 What is an Evidence Statement?
 ------------------------------
-An evidence statement is an assertion about the clinical relevance of a variant that has been determined by an experiment, trial, or study from a published literature source. It captures a variant's impact on clinical action, which can be predictive of therapy, correlated with prognostic outcome, inform disease diagnosis (i.e. cancer type or subtype), or predispose a person to get cancer in the first place. A single citation can be the source of multiple evidence statments, but each evidence statment has only one source. A single evidence statement should correspond to only one clinical interpretation and disease. For example, if a paper describes both a predictive and prognostic relevance for a variant, two evidence statements should be created. If two publications draw the same conclusions about the clinical relevance of a variant, these should also be entered as two evidence statements.
+An evidence statement is a brief description of the clinical relevance of a variant that has been determined by an experiment, trial, or study from a published literature source. It captures a variant's impact on clinical action, which can be predictive of therapy, correlated with prognostic outcome, inform disease diagnosis (i.e. cancer type or subtype), or predict predisposition to cancer in the first place. A single citation can be the source of multiple evidence statments, but each evidence statment has only one source. A single evidence statement should correspond to only one clinical interpretation and disease. For example, if a paper describes both predictive and prognostic relevance for a variant, two evidence statements should be created. If two publications draw the same conclusions about the clinical relevance of a variant, these should also be entered as two evidence statements.
```

Comment:
We still need to consider functional here.

Reply:
added something to this effect